packages/note/src/css/index.js

```diff
@@ -0,0 +1,129 @@
+import core from '@pluralsight/ps-design-system-core'
+import { names as themeNames } from '@pluralsight/ps-design-system-theme/vars'
+
+export const resetButton = {
+  background: 'transparent',
+  border: 'none',
+  color: 'inherit',
+  font: 'inherit',
+  lineHeight: 'normal',
+  margin: 0,
+  overflow: 'visible',
+  padding: 0,
+  width: 'auto',
+
+  MozOsxFontSmoothing: 'inherit',
+  WebkitAppearance: 'none',
+  WebkitFontSmoothing: 'inherit',
+
+  '&::-moz-focus-inner': {
+    border: 0,
+    padding: 0
+  }
+}
+
+export default {
+  '.psds-note__list': {
+    listStyle: 'none',
+    marginLeft: 0
+  },
+
+  '.psds-note': {
+    alignItems: 'flex-start',
+    display: 'flex',
+    fontWeight: core.type.fontWeightMedium,
+    lineHeight: core.type.lineHeightStandard,
+    padding: `${core.layout.spacingLarge} ${core.layout.spacingMedium}`,
+
+    'li &': { borderTop: `1px solid ${core.colors.gray03}` },
+    'li:first-of-type &': { borderTop: 'none' }
+  },
+  [`.psds-note.psds-theme--${themeNames.light}`]: {
+    borderColor: core.colors.gray01
+  },
+
+  '.psds-note__action-bar': {
+    color: core.colors.gray02,
+    display: 'flex',
+
+    'div + &': { marginLeft: core.layout.spacingMedium }
+  },
+  [`.psds-note__action-bar.psds-theme--${themeNames.light}`]: {
+    color: core.colors.gray03
+  },
+
+  '.psds-note__action': {
+    ...resetButton,
+    cursor: 'pointer',
+    fontSize: core.type.fontSizeXSmall,
+    transition: `all ${core.motion.speedNormal}`,
+
+    '&:hover, &:active': { color: core.colors.white },
+    '& + &': { marginLeft: core.layout.spacingSmall }
```

Comment:
For some reason, this is the easiest of all of these nested selectors to read because `&` occupies both the first and second position, making it easy to discern as a "2nd and onward sibling" selector.